data/nlu/nlu.md

```diff
@@ -1,2878 +1,4910 @@
+- difference between rasa [core](product) and [nlu](product)
 - diffrence between rasa [core](product) and rasa [nlu](product)
+- i dont know the difference
 - rasa [core](product) vs rasa [nlu](product)
+- what are the difference between [NLU](product:nlu) and [core](product)?
+- what are the difference?
+- what are the differences?
+- what are the primary areas of difference between [core](product) and [nlu](product)?
+- what differences are there between [core](product) and [nlu](product)?
+- what differences exist between [core](product) and [nlu](product)?
+- what is different
+- what is different about [core](product) compared to [nlu](product)?
+- what is the difference between [core](product) and [nlu](product)
+- what is the difference between [core](product) and [nlu](product)?
+- what is the difference between [nlu](product) and [core](product)
+- what is the difference between [nlu](product) and [core](product)?
+- what is the difference between [nlu](product) and rasa [core](product)
+- what is the difference between rasa [nlu](product) and rasa [core](product)
+- what is the difference between rasa [nlu](product) and rasa [core](product)?
+- what is the difference between the [two](number:2)?
+- what is the difference?
+- what is the different
+- what is the main difference between [core](product) and [nlu](product)?
+- what is the primary difference between [core](product) and [nlu](product)?
+- what makes [core](product) and [nlu](product) unique from each other?
+- what makes [core](product) different from [nlu](product)?
+- what the different with rasa [nlu](product) and rasa [core](product)?
+- what would you say the difference is between [core](product) and [nlu](product)?
+- what's the  difference between rasa [nlu](product) and rasa [core](product)
+- what's the difference between [NLU](product:nlu) and [Core](product:core)?
 - what's the difference between [core](product) and [nlu](product)?
+- what's the difference?
+- whats the diference
+- whats the difference between [core](product) and [nlu](product)
 
 ## intent:ask_faq_differencerasarasax
-- whats the diff between [rasa](product) and [rasa x](product)
-- what's the difference between [rasa](product) and [rasa x](product:x)
-- How is [Rasa X](product:x) different from [Rasa](product:rasa)?
 - Are [Rasa](product) and [Rasa X](product:x) the same thing?
-- What's new in [Rasa X](product:x) compared to [Rasa](product:rasa)?
+- Can I use [Rasa X](product:x) without using [Rasa](product:rasa)?
+- Can i use [rasa](product:rasa) without [rasa x](product:x)?
+- Do I need both [Rasa](product:rasa) and [Rasa X](product:x)?
+- How is [Rasa X](product:x) different from [Rasa](product:rasa)?
 - If I use [Rasa](product:rasa), do I also need [Rasa X](product:x)?
+- What's new in [Rasa X](product:x) compared to [Rasa](product:rasa)?
+- how does [rasa x](product:x) stack up against [rasa](product:rasa)
 - i am using [rasa](rasa) nlu, why would i need [rasa x](product:x)?
```

Comment:
```suggestion
 - i am using [rasa](product) nlu, why would i need [rasa x](product:x)?
```

Reply:
actually this is a weird example bc entity wise it should be nlu, but i think that will make the intent classification weird, best if we probably just remove it

Reply:
Yeah, I changed it to this, to avoid any confusion, and intent classification for the original sentence should be OK.
- i am using [rasa](product), why would i need [rasa x](product:x)?